bazel/README.md

```diff
@@ -1009,5 +1009,5 @@ slower cache performance on macOS due to slow disk performance on Docker for Mac
 Adding the following parameter to Bazel everytime or persist them in `.bazelrc`.
 
 ```
---remote_http_cache=http://127.0.0.1:28080/
+--remote_cache=http://127.0.0.1:28080/
```

Comment:
checking these docs futher - this is the docs for http cache - there are docs  futher up for remote_cache - i think we need to leave this as is

Reply:
The option "remote_http_cache" has been renamed to "remote_cache". We should use the new name.

https://github.com/bazelbuild/bazel/blob/5.2.0/src/main/java/com/google/devtools/build/lib/remote/options/RemoteOptions.java#L114